Ticket1917 camera component

AdrianPotter opened the pull request:
### Description of work

- Made a component type for the Pixelman
- Updated the Pixelman OPI to the new style template

### To test

https://github.com/ISISComputingGroup/IBEX/issues/1917

### Acceptance criteria

As on the original ticket

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/GUI-Coding-Conventions)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit tests in place? Are the unit tests small and test the a class in isolation?
- [ ] Are there [system tests](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/System-Testing-with-RCPTT) in place? Do they test a minimal set of functionality and leave the gui as close as possible to its original state?
- [ ] Did any existing system test break as a result of the current changes? 
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev). If so, do they describe the changes appropriately?
- [ ] Has the [manual system tests spreadsheet](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Manual-system-tests) been updated?

### Functional Tests

- [ ] Do changes function as described? Add comments below that describe the tests performed.
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has developer documentation been updated if required?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

DominicOram commented:
The icon is a little large for my liking (if you add it into the Larmor synoptic you can see it's comparatively bigger than most). Also, can you remove the shadow as no other icons have this.

DominicOram commented:
Also, I'm unsure whether the colour is correct. There is a style guide at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Synoptic-Icons

AdrianPotter commented:
- I took the original colour from the Mercury. Notably it has a different hue than the Eurotherm. I've adjusted the camera hue to match the Eurotherm.
- I've removed the shadow
- I've reduced the size of the icon so it is now a little taller than the standard box icon (e.g. Eurotherm) but significantly narrower.
